components/carbon-authenticators/thrift-authenticator/org.wso2.carbon.identity.authenticator.thrift/src/main/java/org/wso2/carbon/identity/thrift/authentication/internal/ThriftAuthenticatorServiceImpl.java

```diff
@@ -119,8 +122,14 @@ public String authenticate(String userName, String password) throws Authenticati
             // Check if the user is authenticated
             isSuccessful = userRealm.getUserStoreManager().authenticate(tenantLessUsername, password);
 
+            if(isSuccessful){
+                audit.info(String.format(AUDIT_MESSAGE, tenantLessUsername, "SUCCESS"));
+            } else {
+                audit.info(String.format(AUDIT_MESSAGE, tenantLessUsername, "FAILURE"));
+            }
```

Comment:
This will be logged by carbon authentication framework in kernel. Therefore you should not need this log. This may also have the same problem of not logging for error conditions which we may have to fix in kernel.